build.js

```diff
@@ -45,6 +53,7 @@ function buildCSS() {
     .use(require("postcss-inline-svg"))
     .use(require("postcss-css-variables")({ preserve: "computed" }))
     .use(require("postcss-calc"))
+    .use(require("postcss-copy")({ dest: "build" }))
```

Comment:
Could `postcss-url` help us here, with `{ url: 'inline' }`? https://github.com/postcss/postcss-url

Otherwise folks would have to do quite a few network requests to get the font, right? And less portable

Reply:
Inlining addressed, thanks :) https://github.com/jdan/98.css/pull/27#issuecomment-617908646

Can we maintain the original filenames here with the `template` option?